docs/api/README.md

```diff
@@ -61,17 +60,27 @@ The following shows an example of the response in the JSON format:
   "total-service-count": 123,
   "results": [
     {
-      "contact-address": {
-        "street": "Some street 1",
-        "postal_code": "112233",
-        "post_office": "Somewhere"
-      },
       "transport-operator-id": 1,
       "type": "terminal",
       "operator-name": "Some operator, Inc",
-      "name": "The bus stop to somewhere",
+      "name": "The bus stop",
       "sub-type": "terminal",
-      "id": 2
+      "id": 2,
+      "published":"1970-01-01T00:00:00Z",
+      "description":[
+        {"lang":"FI","text":"Bussipysäkki keskellä suomea"}, 
+        {"lang":"EN","text":"Bus stop in the middle of nowhere"}],
+      "business-id":"1234567-0",
+      "external-interface-links":[
+      {"external-interface":
```

Comment:
:url-ote-netex is not mentioned here
but in the other hand it's not in schema either:
https://testi.finap.fi/export/geojson/transport-service.schema.json

But it _is_ in geoJson export, e.g. https://testi.finap.fi/export/geojson/497/1691
I guess we can overlook this for now if you like.

Reply:
👍